useful_extras/create_comparison.py

```diff
@@ -117,6 +117,14 @@ def recreate_single_figure(
                 heights = unyt.unyt_array(this_line_dict["values"], units=plot.y_units)
                 errors = unyt.unyt_array(this_line_dict["scatter"], units=plot.y_units)
 
+                # Data points from the bins with too few data points
+                plot_additional = True
+                try:
+                    additional_x = unyt.unyt_array(this_line_dict["additional_points_x"], units=plot.x_units)
+                    additional_y = unyt.unyt_array(this_line_dict["additional_points_y"], units=plot.y_units)
+                except KeyError:
```

Comment:
This exception enables backwards compatibility. I.e. the script will also be able to plot data from `.yml` files that do have attributes `additional_points_x`, `additional_points_y`. The latter has been the case prior to this update.

Reply:
Yes, I think the best thing to do here though is to have:

```
additional_x = unyt.unyt_array(
    this_line_dict.get("additional_points_x", [])
    units=plot.x_units
)
```